manifests/opensearch-1.1.0.0.yml

```diff
@@ -0,0 +1,51 @@
+---
+schema-version: 1.0
```

Comment:
Isn't this the right schema for manifest.yml as discussed in #111 ? @mch2 @camerski 

```
schema-version: 1.0
product:
  type: opensearch | dashboards
  version: major.minor.patch
  platform: x64 | arm64
components:
  - name: string
    repository: github url
    commit: hash
    original-ref (optional): branch or tag name
  ...
build:
  build-id: string
  location: s3://path/to/<build-id>
```

Reply:
I think it's the right schema, matches 1.0 :)

Reply:
There should be 3 digits in the filename, not 4.

Reply:
I think there's a discrepancy here. The manifest read logic expects a bunch of other things in the manifest file ([code reference](https://github.com/opensearch-project/opensearch-build/blob/main/bundle-workflow/python/manifests/bundle_manifest.py#L12)) and fails if it doesn't discover them.

```
Traceback (most recent call last):
  File "python/test.py", line 17, in <module>
    manifest = BundleManifest.from_file(args.manifest)
  File "/home/ubuntu/cicd/opensearch-build/bundle-workflow/python/manifests/bundle_manifest.py", line 29, in from_file
    return BundleManifest(yaml.safe_load(file))
  File "/home/ubuntu/cicd/opensearch-build/bundle-workflow/python/manifests/bundle_manifest.py", line 35, in __init__
    self.build = self.Build(data['build'])
  File "/home/ubuntu/cicd/opensearch-build/bundle-workflow/python/manifests/bundle_manifest.py", line 51, in __init__
    self.architecture = data['architecture']
KeyError: 'architecture'
```

Reply:
@setiah You're looking at 'BundleManifest'. This is an 'InputManifest'.

Reply:
urgh. my bad! `TooManyManifestsException` :)